### DISCUSSION TITLE
Being transforms politics - or ontology should revolutionise political theory

### DISCUSSION THREAD

**Discussion by rufuspollock** [life-itself/community#1020] [Research]
Claim: "Being transforms politics" - or, more precisely, Ontology revolutionises political theory (and sociology)

Our evolving understanding of psychology, cognitive science, cultural evolution (plus rediscovery of wisdom traditions) (should) revolutionize political theory and sociology.

OR: taking serious account modern ontological sciences - e.g. psychology, cogsci, culturology - would revolutionise sociology and political theory

### 1. By providing a new "end" for politics to pursue

First it suggests a major new *end* for politics: enabling ontological growth personally and collectively -- this is society as "developmental".
 
NB: Even having an aim is sort of controversial now, at least within the democratic liberalism that dominates mainstream political philosophy in the west.[^2] Why, because it implies an end for politics beyond enabling people to do what they want, it implies some kind of "knowing better".

[^2]: todo provide some evidence from e.g. kinds of books being publishing on political theory within university departments

More on this in https://lifeitself.org/ontological-politics. This kind of logic underlies all the "integral politics" / "metamodern" political theory that is out there (e.g. Listening Society and Nordic Ideology). Once you take inner development seriously and its potential for human flourishing and capabilities it naturally becomes a focus for political action. 

### 2. By enlarging the methods of achieving political ends

Second, onto-cultural evolution changes (enlarges) the set of possibilities for coordination and resolving conflict

> Alternatively: It changes the possibility of social organization: as we grow "ontologically" personally and collectively new modes of coordinating become possible.

much of social organization and politics is concerned (explicitly or implictly) with seeking to address collective action problems. relatedly, many political debates revolve around certain fundamental tensions e.g. between individual liberty and collective coordination (cf the debates over covid vaccination). The assumption, often, is that these tensions are inevitable and we can only concern ourselves with either choosing where we make the trade-off (do we trade some individual liberty for better collective outcomes), or structural tweaks to achieve a slight better approach (e.g. participative democracy).

To take one (exaggerated) example: Imagine a group of awakening beings with no ego and total compassion and interbeing. In this group we could solve any collective action problem straight away -- and dispense with most of the structural and technological machinery we need. Put more tersely: a society of enlightened beings would have no tax evasion and would need no tax officers and tax enforcement.

There is an also important aspect in terms of conflict. In a sense politics only begins with some kind of disagreement -- if we are all totally in agreement no discussion is necessary. Ontological and cultural advances can be transformative of how we are able to handle conflict: for example in our capacity to take multiple perspectives including that of those we disagreement with, to hold our own views lightly ([[non-attachment to views]]). Often, much political disagreement isn't just about principles but about "facts". An ability to find coherence and to collectively make sense would make a huge difference in resolving social and political conflict. To come back to an example of covid vaccinations. Often the debate over principle and the debate over facts become interwoven: e.g. objections to being "compelled" to vaccinate (e.g. to lose your job as a nurse if you don't) is intermixed with questions of whether a vaccination is effective and/or dangerous.

### 3. By providing a new and fundamental framing for analysing political thought

Third, in terms of political theory and political thought, ontology and ontological context provides a novel foundational frame with which to understand the evolution of this thought. put another way, most political thought, and most *history* of political thought exists without awareness of ontological/cultural context. How so?

First, whilst debates about the nature of human nature do arise (e.g. classic Hobbes vs Rousseau), they are not seen as central and as a general organizing axis for interpreting and assessing political thought[^1]. Furthermore, they are not seen as claims that could be subject to scientific analysis -- we don't have section in political textbooks from cognitive scientists evaluating Rousseau's claims about human nature (though note that for his time Rousseau was something of an ontologist writing books not just on political theory but on children's education)

Second, and perhaps even more importantly, almost all political thinkers -- and all historians of those thinkers -- assume a **fixed** human nature (even if there are debates over the nature of human nature cf classic Hobbes vs Rousseau). This points of course, links back to the first about the *aim* of politics.

[^1]: what i mean by that is you don't open a textbook on political theory and see for each thinker a subsection called "ontology" or similar. It is not a major organizing dimension for how we assess or compare thinkers.

## Ideas for Next Steps

- [ ] A reader of major social/political thinkers "ontology" with modern science critique. Bonus: add a section of their social/political views.
  - [ ] e.g. Montaigne has an essay on education (and even wise education IIRC)
  - [ ] Rousseau
  - [ ] Hobbes
  - [ ] Plato
  - [ ] Lao-Tzu
  - [ ] Confucius (and followers)
  - [ ] Marx
  - [ ] Hegel?
- [ ] Summary table of their ontological views and their political prescriptions

**Comment by matthewmccarthy11** [comment]
Putting this here as a potential example as it relates to point 1: https://www.sarvodaya.org/
https://en.wikipedia.org/wiki/Sarvodaya 

Mission statement is "to build a just, sustainable, compassionate social order that fulfils the basic human needs of the community through individual and collective awakening. "

They seem also to be split into different 'units' https://www.sarvodaya.org/units/ 

Seems like this would maybe be a 'political model' which relates to enabling ontological growth.

"The Sarvodaya Shramadana Movement of Sri Lanka consists of the main incorporated body officially known as Lanka Jathika Sarvodaya Shramadana Sangamaya (Inc.), a group of legally independent units catering to specific fields of development activities and the village level Sarvodaya Shramadana Societies (SSSs). In each SSS there are child members (aged 7-14), youth members (14-25), ordinary members (over 25 years), life members, and invited members (honorary members). From this is drawn an executive council of 25 members, 18 executive members (3 children, 3 youth, 3 mothers and 9 ordinary members) and 7 elders (selected from life and ordinary members). Sub-committees help to fulfil the needs of the village."

**Comment by rufuspollock** [comment]
MOVED to https://forum.secondrenaissance.net/t/being-transforms-politics-or-taking-ontolo-genesis-culturo-genesis-seriously-would-revolutionize-political-theory/366
